terraform/install.sh

```diff
@@ -59,6 +57,45 @@ loadGen() {
   ../loadgenerator-tool startup --zone us-west2-a $external_ip
 }
 
+displaySuccessMessage() {
+    created_project=$(cat ./terraform.tfstate | \
+                        grep "\"project\":" | \
+                        grep -oh "stackdriver-sandbox-[0-9]*" | \
+                        head -n 1)
+
+    gcp_path="https://console.cloud.google.com/kubernetes/workload"
+    if [[ -n "${created_project}" ]]; then
+        gcp_path="$gcp_path?project=$created_project"
+    fi
+    TRIES=0
+    while [[ -z "${loadgen_ip}" && "${TRIES}" -lt 20  ]]; do
+        sleep 1
+        loadgen_ip=$(gcloud compute instances list --project "$created_project" \
+                                                   --filter="name:loadgenerator*" \
```

Comment:
Worth adding log messages - mainly for potential troubleshooting if something will happen in the future